### PR TITLE
Update libcontainer to fd0087d3acdc4c5865de1829d4accee5e3ebb658

### PR DESCRIPTION
**Pull request by crosbymichael**
Signed-off-by: Michael Crosby crosbymichael@gmail.com

### CODE REVIEW

File: hack/vendor.sh [hunk 1]
```diff
@@ -75,7 +75,7 @@ rm -rf src/github.com/docker/distribution
 mkdir -p src/github.com/docker/distribution
 mv tmp-digest src/github.com/docker/distribution/digest
 
-clone git github.com/docker/libcontainer 4a72e540feb67091156b907c4700e580a99f5a9d
+clone git github.com/docker/libcontainer fd0087d3acdc4c5865de1829d4accee5e3ebb658
```
Comment (LK4D4) [inline hack/vendor.sh]: commit is different from commit message

Comment (crosbymichael) [inline hack/vendor.sh]: fixed

**Comment by jfrazelle** [comment]
LGTM

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**